CactbotOverlay/VersionChecker.cs

```diff
@@ -52,37 +53,22 @@ class VersionChecker {
     }
 
     public Version GetRemoteVersion() {
-      string html;
       try {
-        var web = new System.Net.WebClient();
-        System.Net.ServicePointManager.SecurityProtocol = System.Net.SecurityProtocolType.Ssl3 | System.Net.SecurityProtocolType.Tls | System.Net.SecurityProtocolType.Tls11 | System.Net.SecurityProtocolType.Tls12;
-        var page_stream = web.OpenRead(kReleaseUrl);
-        var reader = new System.IO.StreamReader(page_stream);
-        html = reader.ReadToEnd();
+        string json;
+        using (var webClient = new System.Net.WebClient()) {
+          // https://developer.github.com/v3/#user-agent-required
+          webClient.Headers.Add("User-Agent", "Cactbot");
```

Comment:
Sorry for style nit, but `web_client` instead of `webClient` to match the other cactbot C# code? (and elsewhere)